vernac/comAssumption.ml

```diff
@@ -198,8 +196,8 @@ let context_insection sigma ~poly ctx =
         declare_variable false ~kind t [] impl (CAst.make name)
       | name, Some b, t, impl ->
         (* We need to get poly right for check_same_poly *)
```

Comment:
While you're at it please change the comment to `univs ignored with scope:Discharge` or equivalent comment.

Reply:
Actually I'm not confident if that's true anymore, I don't remember exactly what we do in this case.

Reply:
I'm confused. There are situations where would we like to ignore universes?